src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.AltSvc.cs

```diff
@@ -71,9 +73,14 @@ fromVersion.Major switch
                 { HttpVersion.Version20, false }
             };
 
-        [Fact]
+        [ConditionalFact]
         public async Task AltSvc_ConnectionFrame_UpgradeFrom20_Success()
```

Comment:
I believe this one is always about H/2 -> H/3 upgrade, so this should be just `[ActiveIssue]`